src/components/core/Headline.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import noop = require('lodash/noop');
+import { HeadlineTransformation } from '@gooddata/indigo-visualizations';
+import { Execution } from '@gooddata/typings';
+
+import { IntlWrapper } from './base/IntlWrapper';
+import { IntlTranslationsProvider, ITranslationsComponentProps } from './base/TranslationsProvider';
```

Comment:
This is just a suggestion but I would rather put each imported variable on the new line when there are multiple imports (same goes for line 13 and few other changes files as well).
 ```
 import { 
   IntlTranslationsProvider, 
   ITranslationsComponentProps 
 } from './base/TranslationsProvider';
```